chia/consensus/full_block_to_block_record.py

```diff
@@ -88,6 +88,13 @@ def block_to_block_record(
     )
 
 
+def maybe_uint64(val: Optional[int]) -> Optional[uint64]:
```

Comment:
Maybe add `StructStream.maybe()` instead.

Reply:
how would that work?

Reply:
```python
@classmethod
def maybe(cls: Type[T], val: Optional[int]) -> Optional[T]:
    # regular logic here
```

Reply:
I think `uint64.maybe()` reads a bit unintuitively. I called it `construct_optional()`